### DISCUSSION TITLE
so-preflight

### DISCUSSION THREAD

**Discussion by JohonJ** [Security-Onion-Solutions/securityonion#7704] [Unsupported Versions]
during the installation of 2.3.110.iso  for so-import on VirtualBox, I received an error _**that the so-preflight script failed to check one or more URLs required by setup**_. macOS is the main OS on the laptop.
Do I miss something?

**Comment by dougburks** [comment]
Does your virtual machine have Internet access?

Have you double-checked your IP address, default gateway, and DNS server settings?

Have you checked `/root/sosetup.log` for additional clues?

**Reply by faizans986** [reply]
Hello @dougburks 

Same issue happens with me , I had checked preflight.log file and found below pasted error.
![image](https://user-images.githubusercontent.com/67871496/164800580-86e2fed7-ddc6-439d-b3f0-551c2013c149.png)
Thanks in advance for your help.

**Reply by dougburks** [reply]
@faizans986 From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
